src/core/gameworld.ts

```diff
@@ -555,7 +562,11 @@ export default class GameWorld {
             const didMove = this.npcMoveRandom(this.getNpcByUniqueId(npc.uniqueId));
             if (didMove) {
                 this.getNpcByUniqueId(npc.uniqueId).lastMoveTime = new Date();
-                // console.log('[WORLD] handleNpcTurn', npc.uniqueId, world.getNpcByUniqueId(npc.uniqueId).lastMoveTime);
+                this.logger.trace(
+                    '[WORLD] handleNpcTurn',
```

Comment:
Just wondering, since each class seems to have its own instance of the logger, is there a way to have them use different prefixes? Like gameworld could have `[WORLD]` automatically prepended to every log.

Otherwise, I saw there's a way to declare children loggers, so what do you think about having one main logger declared and exported in a separate file and each class instanciates a `logger.child({ module: 'world' })` for example